test/transforms/cypress-plugins.test.js

```diff
@@ -2,7 +2,8 @@ import expect from 'expect';
 import applyTransform, { dedent } from '../helpers/apply-transform';
 import transform from '../../transforms/cypress-plugins';
 
-describe('Transforms - cypress-plugins.js', () => {
+describe('Transforms - cypress-plugins.js', function() {
+  this.timeout(6000);
```

Comment:
Should we just set a global 10s timeout in the package.json mocha settings? That's usually my go-to for solving flakey Windows timeouts since it's entirely possible for this to crop up in another transform test at some point